projects/plugins/jetpack/extensions/blocks/subscriptions/settings.js

```diff
@@ -66,56 +86,109 @@ export function NewsletterAccess( { accessLevel, setPostMeta, withModal = true }
 	}
 	const accessLabel = accessOptions[ accessLevel ]?.label;
 
+	// Can be “private”, “password”, or “public”.
+	const postVisibility = useSelect( select => select( 'core/editor' ).getEditedPostVisibility() );
+	const postVisibilityIsPublic = postVisibility === 'public';
+
+	const showVisibilityRestrictedMessage = ! postVisibilityIsPublic && accessLevel === 'everybody';
+	const showMisconfigurationMessage = ! postVisibilityIsPublic && accessLevel !== 'everybody';
+
 	return (
 		<PostVisibilityCheck
 			render={ ( { canEdit } ) => (
 				<PanelRow className="edit-post-post-visibility">
-					<Flex direction={ withModal ? 'row' : 'column' }>
-						{
+					<Flex direction={ 'column' }>
+						<FlexBlock>
+							<InspectorNotice spanClass={ 'jetpack-subscribe-info' }>
+								{ createInterpolateElement(
+									/* translators: basic information about the newsletter visibility */
```

Comment:
A few optional thoughts in regard to the informational message:

1. Instead of displaying the info message in the panel. Could we add an info icon to the right of the newsletter header and use a [Popover](https://wordpress.github.io/gutenberg/?path=/story/components-popover--default&args=shift:false;resize:false;position:top+left;focusOnMount:true) component to display the message? I think it would be nice if the popover shows on hover as well

2. What do you think about changing the wording of the message to:  "Restrict your post to subscribers. Learn more" ( learn more would be the link )

Reply:
> Instead of displaying the info message in the panel. Could we add an info icon to the right of the newsletter header and use a Popover

I do agree it could be better, and could be something for a new task. I also wonder if there is a standard way in FSE to do that...

> What do you think about changing the wording of the message to: "Restrict your post to subscribers. Learn more" ( learn more would be the link )

Nice.

Reply:
Done

Reply:
<img width="269" alt="Screenshot 2023-02-24 at 10 58 01" src="https://user-images.githubusercontent.com/790558/221149435-3398f715-0039-4a89-b0f7-c8dbe200db2e.png">

I like it, even if it is confusing with the "newsletter" name..